src/reducers.js

```diff
@@ -5,12 +5,10 @@
 
 import { combineReducers } from 'redux';
 
-import { reducer as navigation } from './navigation';
-import { reducer as sync } from './sync';
+import { reducer as appReducer } from './navigation';
+import { reducer as syncReducer } from './sync';
 
-export const reducers = combineReducers({
-  navigation,
-  sync,
+export default combineReducers({
+  nav: appReducer,
+  sync: syncReducer,
```

Comment:
seems this is just renamed to `syncState` when it is used? Should we just name it that here?

Reply:
@joshxg They are state of the store, so they can be named syncState but that would become state.syncState which is not desirable either. One can even argue that after mapStateToProps they become props. :D

I would have preferred just naming them without `state` as word, but navigation is taken by react navigation plugin and only renaming sync does not sound too consistent.